common/helpers/src/serde.rs

```diff
@@ -0,0 +1,27 @@
+// use serde_json_core::{Result, Value, from_slice};
+use serde_json::{Result, Value, from_slice};
+
+pub fn validate_json_schema(json_schema: Vec<u8>) -> Result<()> { // short curcuit
+    let result: Value = from_slice(&json_schema)?;
```

Comment:
Does this support json with UTF-8 input? Maybe we need a test with UTF-8 data to check.

Reply:
So I would test with Json with utf-8 encoding, and the get-schema-id should return the proper schema right?